GitHubAssignees.ps1

```diff
@@ -359,7 +367,9 @@ function Remove-GithubAssignee
 
         [string] $AccessToken,
 
-        [switch] $NoStatus
+        [switch] $NoStatus,
+
+        [switch] $Force
```

Comment:
`AccessToken` and `NoStatus` are normally the last parameters on each function. How about moving the `Force` parameter to before them to keep consistency across all functions? Same on all functions if you want to make the change.

Reply:
@X-Guardian I've re-ordered them 😄